src/utils/resolveLatestVersion.js

```diff
@@ -0,0 +1,26 @@
+/** @flow */
+/**
+ * Retrieve latest bit version from latest to number
```

Comment:
This description is not precise.
Change it to Retrive bitId with the highest version from a list according to provided id.
(add to description that it will return the provided id if it's not with latest, also add that if the list contain id without version / with the latest version it will return it)

Also fill the args docs, especially the fact that we support both strings and ids.
About the return values, it's not always a string.. actually i think it will be better to get it as BitId always (just call the getParsed for the result).